app/components/signup.tsx

```diff
@@ -0,0 +1,131 @@
+'use client'
+import Image from 'next/image'
+import Link from 'next/link'
+import { useState, FormEvent } from 'react'
+import profile from './profile.svg'
+export default function Signup_Form() {
+  const [email, setEmail] = useState<string>('')
+  const [password, setPassword] = useState<string>('')
+  const [confirmPassword, setConfirmPassword] = useState<string>('')
+
+  const handleEmailChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setEmail(event.target.value)
+  }
+
+  const handlePasswordChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setPassword(event.target.value)
+  }
+  const handleConfirmPasswordChange = (
+    event: React.ChangeEvent<HTMLInputElement>,
+  ) => {
+    setConfirmPassword(event.target.value)
+  }
+  const handleSubmit = (event: FormEvent) => {
+    event.preventDefault()
+    if (password !== confirmPassword) {
+      alert('Passwords do not match')
```

Comment:
@mohd-khan09 let's not have alerts for our error handling, you can create a state to store and show errors in UI. For eg. Use conditionally rendered span tag to show error below the input boxes.